latex.tplx

```diff
@@ -1,3 +1,28 @@
+%=====================================
+% Solution about utf8x problem
+% https://github.com/jupyter/nbconvert/issues/530#issuecomment-303034557
+%=====================================
+
+((*- extends 'article.tplx' -*))
+[unrelated template changes]
```

Comment:
I think this line was just an example in the suggested workaround, so we don't need it!